### ISSUE TITLE
Fix the LICENSE file contents, so that package docs show up properly on pkg.go.dev

### ISSUE THREAD

**Issue by colakong** [imgix/prometheus-am-executor#25]
README and documentation for this project [doesn't show up in pkg.go.dev](https://pkg.go.dev/github.com/imgix/prometheus-am-executor?tab=overview). The reason given is that the license file doesn't match the golang package directory website's license policy.

This issue is to determine what needs to change in our LICENSE file, in order for README/Docs to show up properly in pkg.go.dev.

Some helpful links for this issue:

* [pkg.go.dev license policy](https://pkg.go.dev/license-policy)
* The OSI formatting of the [BSD 2-clause license](https://opensource.org/licenses/BSD-2-Clause) that we're using (which is detected by GitHub)

**Comment by colakong** [comment]
I used the google [licensecheck](https://pkg.go.dev/github.com/google/licensecheck?tab=doc) package to check the license of prometheus-am-executor, and it was detected correctly as BSD 2-clause.

```
./check_license ~/work/src/github.com/imgix/prometheus-am-executor/LICENSE
licensecheck.Coverage{Percent:100, Match:[]licensecheck.Match{licensecheck.Match{Name:"BSD-2-Clause", Type:2, Percent:100, Start:0, End:1301, IsURL:false}}}
```

I put the license checking code [in its own repo](https://github.com/colakong/check_license) with the same license as prometheu-am-executor ((c) me instead of Zebrafish Labs though), and [pkg.go.dev detected it correctly](https://pkg.go.dev/github.com/colakong/check_license?tab=overview).

The only difference between the two licenses, is the copyright holder name. This difference means that my license file is 1300 bytes, and the prometheus-am-executor one is 1301 bytes.

From what I can tell, the issue seems to be that pkg.go.dev hasn't re-crawled the repository since the LICENSE file was added 15 months ago.

**Comment by otakup0pe** [comment]
Hmm that seems suspicious, I've definitely seen BSD licenses with with multiple copyright holders in golang projects (unsure as to whether they are detected by pkg.go.dev tho)

**Comment by colakong** [comment]
Fixed. There were two things I did.
1. Fix the formatting of the `go.mod` file; The module name should have been the full path (2d55165175ca8174a4514c1870727460c76ec0b7)
This allows commands like `go get github.com/imgix/prometheus-am-executor` to work, even though this isn't a module-type package.

2. Ask golang tooling to fetch the newest version of the package. I did this in two ways:
* Ask the checksum database for the newest version
```
curl https://sum.golang.org/lookup/github.com/imgix/prometheus-am-executor@v0.1.1
```
* Ask pkg.go.dev to specifically fetch the new version
```
curl -X POST --include 'https://pkg.go.dev/fetch/github.com/imgix/prometheus-am-executor@v0.1.1'
```

Between these two items, and the two requests that were made, pkg.go.dev now shows the license and README for the package :)
